honeycomb/honeycomb_reporter.go

```diff
@@ -30,8 +31,9 @@ func New(client client.Client) honeyCombReporter {
 
 func (hr honeyCombReporter) SpecDidComplete(specSummary *types.SpecSummary) {
 	specEvent := SpecEvent{
-		State:       getTestState(specSummary.State),
-		Description: createTestDescription(specSummary.ComponentTexts),
+		State:            getTestState(specSummary.State),
+		Description:      createTestDescription(specSummary.ComponentTexts),
+		RunTimeInSeconds: int(specSummary.RunTime / 1000000000),
```

Comment:
Any reason you're not using the built-in time.Duration methods for getting seconds and/or nanoseconds? (https://golang.org/pkg/time/#Duration.Seconds, https://golang.org/pkg/time/#Duration.Nanoseconds)

Reply:
Didn't see that when I was looking at the time package!

Also, Stack Overflow suggested doing it this way.

Never listen to Stack Overflow.